### PR TITLE
24-Feat/add-schema-file-and-users-table-Leila

### PR DESCRIPTION
**Pull request by leilafarsani**
This is a:

- [x] ✨ **New feature** - new behaviour has been implemented
- [ ] 🐛 **Bug fix** - existing behaviour has been made to behave
- [ ] ♻️ **Refactor** - the behaviour has not changed, just the implementation
- [ ] ⚙️ **Chore** - maintenance task, behaviour and implementation haven't changed

### Description

Description:
In this pull request, we introduce the initial setup for the "users" table and its associated schema definition. The changes include the creation of the "users_schema.sql" file, which contains the necessary SQL code to create the table with appropriate columns.

The SQL code in "users_schema.sql" has been tested and executed successfully in a local PostgreSQL database.
IMAGE 2023-07-27 22:53:20
### How to test:

- connect to the database 
- Type `SELECT * FROM users; ` or ` \d users` in the terminal
- check the the users table and its columns.

### Related to

Feat #24 

### Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have carefully reviewed my own code
- [ ] I have commented my code
- [X] I have updated any documentation
- [X] I have written a title that reflects the relevant ticket
- [X] I have written a description that says what the PR does and how to validate it
- [X] I have linked to the project board ticket (and any related PRs/issues) in the Links section
- [X] I have added a link to this PR to the ticket
- [ ] I have requested reviewers here and in my team chat channel

### CODE REVIEW

File: server/schema/users_schema.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    role VARCHAR(50) NOT NULL,
+    nationality VARCHAR(50),
+    location VARCHAR(100) NOT NULL,
+    soft_skills VARCHAR(200),
+    hard_skills VARCHAR(200),
+    mobile VARCHAR(20),
+    email VARCHAR(100) NOT NULL,
+    linkedin VARCHAR(100),
+    youtube VARCHAR(100),
+    description VARCHAR(500) NOT NULL,
```
Comment (Ali-Jahankah) [inline server/schema/users_schema.sql:18]: No more than 255

File: server/schema/users_schema.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    hobbies VARCHAR(200)
+);
+
+
+
```
Comment (Ali-Jahankah) [inline server/schema/users_schema.sql:23]: Nice work but Better to set default values for optional fields to '__' for writing less error handling codes in Front End and showing in the app. So just use DEFAULT and give them default values.

**Review by Ali-Jahankah — CHANGES_REQUESTED** [review]

File: server/schema/users_schema.sql (hunk 2 above)
Comment (leilafarsani) [reply server/schema/users_schema.sql:22]: Many thanks for the review! Done!

**Comment by Ali-Jahankah** [comment]
Nice work!

**Pull request merged**